internal/cmd/db_list.go

```diff
@@ -8,8 +8,13 @@ import (
 	"golang.org/x/exp/slices"
 )
 
+var groupFilter string
+var schemaFilter string
+
 func init() {
 	dbCmd.AddCommand(listCmd)
+	listCmd.Flags().StringVarP(&groupFilter, "group", "g", "", "Filter databases by group")
```

Comment:
For all new flags you create, use the flags package. I'll make a PR adding the group and schema flags there and add you as a reviewer.